libs/sdk-ui-ext/src/internal/components/configurationControls/colors/colorDropdown/tests/ColorDropdown.test.tsx

```diff
@@ -24,76 +26,87 @@ const defaultProps: IColorDropdownOwnProps = {
     onColorSelected: noop,
 };
 
-class MockItem extends React.PureComponent<IColoredItemContentProps> {
+class MockItem extends React.PureComponent<IMockItemProps> {
     public render() {
-        return <button className="buttonitem">test</button>;
+        return (
+            <button data-is-selected={this.props.isSelected} data-icon-position={this.props.position}>
+                test
+            </button>
+        );
     }
 }
 
 function createComponent(customProps: Partial<IColorDropdownOwnProps> = {}) {
     const props: IColorDropdownOwnProps = { ...cloneDeep(defaultProps), ...customProps };
-    return mount<IColorDropdownOwnProps>(
-        <InternalIntlWrapper>
-            <ColorDropdown {...props}>
-                <MockItem color={{ r: 255, g: 0, b: 0 }} text="sometext" />
-            </ColorDropdown>
-        </InternalIntlWrapper>,
-    );
+
+    return {
+        user: userEvent.setup(),
+        ...render(
```

Comment:
Do we have some testing utilities in some project that every project can read from? We could define `render` method there that would take react component and wrap this userEvent setup. I'm not sure how complicated would this method be to type, if there are some more parameters and that are needed to be provided from time to time, etc. Just a suggestion as this seems to be some kind of test bootstrap that is being copied from test file to test file.

````
import { render } from "my/testing/utils";

function createComponent(customProps: Partial<IColorDropdownOwnProps> = {}) {
    const props: IColorDropdownOwnProps = { ...cloneDeep(defaultProps), ...customProps };
    return render(
    <InternalIntlWrapper>
          <ColorDropdown {...props}>
                 <MockItem color={{ r: 255, g: 0, b: 0 }} text="sometext" />
          </ColorDropdown>
   </InternalIntlWrapper>,
);

...
import { render } from testing-library;

const render = (component: React.Component) => {
 return {
	user: userEvent.setup(),
        ...render(component),
  };
}
````

Reply:
Yeah, this is something I would like to have. Probably need to talk to Matyas about best place for such function.